model_analyzer/config/input/config_command_analyze.py

```diff
@@ -414,13 +414,21 @@ def _autofill_values(self):
                 new_model['objectives'] = self.objectives
             else:
                 new_model['objectives'] = model.objectives()
+
             # Constraints
             if not model.constraints():
                 if 'constraints' in self._fields and self.constraints:
                     new_model['constraints'] = self.constraints
             else:
                 new_model['constraints'] = model.constraints()
 
+            # Weightings
```

Comment:
do we still need to update both ConfigAnalyze and ConfigProfile? Also, why is the newly added code different in each?

Reply:
We do. And, this is just one(more) reason why we should get rid of analyze. This code should be identical to profile. Good catch.